scripts/plugins/extauth-hook-AD.py

```diff
@@ -248,6 +260,10 @@ def __init__(self, session, args, ad_enabled=True):
         self._update_key_value("GSSAPIAuthentication", value)
         self._update_key_value("GSSAPICleanupCredentials", value)
 
+    def apply(self):
+        super(SshdConfig, self).apply()
+        run_cmd(["/usr/bin/systemctl", "restart", "sshd"])
```

Comment:
SSHd is smart enough to keep existing connections open over a restart (it uses a child process), but if all you want is to load a new config then `reload` might be better here

Reply:
Indeed, `sshd.service` supports `reload-or-restart` which would use `restart` as a fallback if it were not supported.